### PR TITLE
Now campaign plots have an alpha

### PR DESCRIPTION
**Pull request by christinahedges**
And kwargs is a better way to deal with the matplotlib styling.

### CODE REVIEW

**Comment by christinahedges** [comment]
@barentsen plz check this out and merge when ready?

File: K2fov/data/k2-campaign-parameters.json [hunk 1]
```diff
@@ -206,5 +206,15 @@
...
     "stop": "2018-10-10",
     "comments": "Forward-facing; Trappist-1, Neptune."
+  },
+
+  "c20": {
+   "campaign": 20,
+   "ra": 71.2012890,
+   "dec": 23.1361165,
+   "roll": 7.9176424,
+   "start": "2018-10-11",
```
Comment (barentsen) [inline K2fov/data/k2-campaign-parameters.json:216]: Let's make this `"start": "2018-10-15"` for now.

**Review by barentsen — CHANGES_REQUESTED** [review]

File: K2fov/fields.py [hunk 2]
```diff
@@ -64,7 +64,7 @@ def getFieldInfo(fieldnum):
                          "of the code".format(fieldnum))
 
 
-def getKeplerFov(fieldnum):
+def getKeplerFov(fieldnum, broken=True):
```
Comment (barentsen) [inline K2fov/fields.py:67]: The `broken` parameter is not used in the function, I am going to remove it?

Comment (christinahedges) [reply K2fov/fields.py:67]: I used it for plotting the C20 stuff so we could see a module under mars. You can remove it.

File: K2fov/data/k2-campaign-parameters.json (hunk 1 above)
Comment (christinahedges) [reply K2fov/data/k2-campaign-parameters.json:216]: Sure

Comment (barentsen) [reply K2fov/data/k2-campaign-parameters.json:216]: Actually: this looks like the wrong field to me.  Let's make sure we use the correct ra/dec/roll parameters for option 1 here.

**Comment by christinahedges** [comment]
@barentsen I DID A BAD THING

**Pull request merged**